### ISSUE TITLE
Upgrade to newest roundcube

### ISSUE THREAD

**Issue by sirtimbly** [mail-in-a-box/mailinabox#839]
Just putting this here so I know when we can get Roundcube upgraded. The latest release supports PGP through Mailvelope https://roundcube.net/news/2016/05/22/roundcube-webmail-1.2.0-released

**Comment by aspdye** [comment]
I will do this later today :wink:

**Comment by jkaberg** [comment]
Let it be [known](https://github.com/roundcube/roundcubemail/blob/master/plugins/enigma/README)

**Comment by aspdye** [comment]
Will be fixed with #840 😉

**Comment by jkaberg** [comment]
The point of my comment was to point out that the PGP part isn't feature complete. I'd hold of on this one for a while.

From README @ roundcube
```
[...]

TODO:
-----
- Handling of big messages with temp files
- Key info in contact details page (optional)
- Extended key management:
   - disable,
   - revoke,
   - change expiration date, change passphrase, add photo,
   - manage user IDs
   - export private keys
- Generate revocation certs
- Search filter to see invalid/expired keys
- Key server(s) support (search, import, upload, refresh)
- Mark keys as trusted/untrasted, display appropriate message in verify/decrypt status
- Change attachment icon on messages list for encrypted messages (like vcard_attachment plugin does)
- Support for multi-server installations (store keys in sql database?)
- Per-Identity settings (including keys/certs)
- Performance improvements:
   - cache decrypted message key id so we can skip decryption if we have no password in session
   - cache (last or successful only?) sig verification status to not verify on every msg preview (optional)
- S/MIME: Certs generation
- S/MIME: Certs management
- S/MIME: signed messages verification
- S/MIME: encrypted messages decryption
- S/MIME: Sending signed/encrypted messages
- S/MIME: Handling of certs attached to incoming messages
- S/MIME: Certificate info in Contacts details page (optional)


KNOWN ISSUES:
-------------
There are some know issues with accepting key passphrases on various
system configurations. This is caused by issues in PinEntry handling.
Make sure that vendor/bin/crypt-gpg-pinentry works from command line.
Possible reasons:
- non-supported GnuPG version, i.e. >= 2.1
- non-working loader in shebang (#! /usr/bin/env php)
```

**Comment by sirtimbly** [comment]
Yeah, storing the keys on the server and server side decryption is a dubious plan anyway. But, just supporting mailvelope out of the box is a nice upgrade.

**Comment by JoshData** [comment]
Fixed by #840 (thanks @aspdye).
